### DISCUSSION TITLE
having issues while install framer-motion for next 15

### DISCUSSION THREAD

**Discussion by waleed-taher** [motiondivision/motion#2839] [Q&A]
I have upgraded to next 15 stable version with react 19-rc and having issues while add framer motion to my project so if anyone of you can help out it would be really great thanks

![image](https://github.com/user-attachments/assets/807401a4-8e0f-4354-bbc4-4e336b76d9dd)

**Comment by iraritchiemeek** [comment]
It looks like the `package.json` hasn't been updated to reflect the react 19 support yet.

Installing with the following flag should let you get around this for now:
`npm install framer-motion@12.0.0-alpha.0 --legacy-peer-deps`

**Reply by Babailan** [reply]
why don't they change it to ^19 in the documentation they say it's working already

**Reply by jthrilly** [reply]
React 19 support is WIP: https://github.com/motiondivision/motion/pull/2667 and the current React 19 version is `12.0.0-alpha.2`
